Source/Urho3D/Physics/CollisionShape.cpp

```diff
@@ -1037,19 +1022,18 @@ void CollisionShape::UpdateShape()
 
     if (node_)
     {
-        Scene* scene = GetScene();
-        Vector3 newWorldScale = node_->GetWorldScale();
+        cachedWorldScale_ = node_->GetWorldScale();

```

Comment:
cachedWorldScale_ is (and was) used to check whether it is needed to recreate the shape.
I just moved it above to avoid redundand local variable and simplify extracted code.